dataplatform_sync/settings/base.py

```
@@ -106,26 +106,45 @@
 CELERY_RESULT_BACKEND = os.environ.get(
     'CELERY_RESULT_BACKEND', 'redis://localhost:6379/0')
 
+MATILLION_INSTANCE_ID = os.environ.get(
+    'MATILLION_INSTANCE_ID', '')
```

I would say this is mandatory and should fail if not specified?